cli/docs/cli.json

```diff
@@ -79,8 +84,8 @@
       "subcommands": [
         {
           "name": "login",
-          "about": "Authenticate with an Oxide host.",
-          "long_about": "Authenticate with an Oxide host.\n\nAlternatively, pass in a token on standard input by using `--with-token`.\n\n    # start interactive setup\n    $ oxide auth login\n\n    # authenticate against a specific Oxide instance by reading the token\n    # from a file\n    $ oxide auth login --with-token --host oxide.internal < mytoken.txt\n\n    # authenticate with a specific Oxide instance\n    $ oxide auth login --host oxide.internal\n\n    # authenticate with an insecure Oxide instance (not recommended)\n    $ oxide auth login --host http://oxide.internal",
+          "about": "Authenticate with an Oxide Silo",
```

Comment:
I think Silo is a term that we're educating users about. Certainly it's more precise than a "host" ... which is a term of whose definition I'm very unsure.

Reply:
Yeah, this is a tough one. In the web console, we generally say that for the developer user without fleet permissions, the silo is their universe and we try to avoid talking about the silo. However, we do say it right in the top left corner, and (as of recently) we call silo images "silo images". So it's not completely hidden.

<img width="852" alt="image" src="https://github.com/user-attachments/assets/d57093ba-110a-41fe-a627-c8065575abe2">

Notably silos are also the first thing in the key concepts guide. So I think we've definitely softened the "first rule of silos is never talk about silos" idea.

https://docs.oxide.computer/guides/key-entities-and-concepts